packages/commons-ui-core/src/styles/createTheme/createContentWidths.js

```diff
@@ -0,0 +1,37 @@
+function createContentWidths(contentWidths, breakpoints) {
+  if (contentWidths) {
+    return contentWidths;
+  }
+  if (breakpoints) {
+    // Use https://mui.com/customization/breakpoints/#default-breakpoints
+    // as is for content widths i.e. body
+    const { values: breakpointValues, unit } = breakpoints;
+    const values = { ...breakpointValues };
+    // Assume values are sorted, remove the first value so that content width
+    // is 100% of the size.
+    delete values[Object.keys(breakpointValues)[0]];
```

Comment:
@kilemensi by sorted, do you mean sorted alphabetically? or sorted per screensize? 

If `xs` is what we need to delete, why not perform delete by just using `xs` key

Reply:
By screen size @KhadijaMahanga ... and MUI allows [completely](https://mui.com/material-ui/customization/breakpoints/#custom-breakpoints) customization of key names so we can't assume it'll be always `xs`.